mosaic-site/docs/getting-started/developing.mdx

```diff
@@ -149,7 +149,7 @@ function App() {
 export default App;
 ```
 
-Head over to our available [components](../components/) to view their respective APIs.
+Head over to our available [components](../components/index) to view their respective APIs.
```

Comment:
I expect the URLs of our site's pages not to change as part of the Mosaic migration.

Can Mosaic not be configured so that the path `/components/` points to the page with the status table, just as it does on the current Docusaurus site?

Then these links wouldn't need changing as well.